Add CloudPrem installation page with docker and docker compose. (#31492)
DataDog/documentation: content/en/cloudprem/architecture.md

**Janitor**
: Performs maintenance tasks, applying retention policies, garbage collecting expired splits, and running delete query jobs.

## Connection to Datadog UI

> There are two ways to use connect Datadog UI to CloudPrem:

Reviewer: ```suggestion
There are two ways to connect the Datadog UI to CloudPrem:
```

> - Accept external requests from Datadog: You give Datadog a DNS to send gRPC requests to, configure an Public Ingress to accept those requests. Learn how to configure the ingress [here]()

Reviewer: ```suggestion
- **Accept external requests from Datadog**: Provide Datadog with a DNS endpoint for gRPC requests and configure a public Ingress toaccept those requests. Learn [how to configure the ingress][].
```

> - **Reverse connection**: Let CloudPrem initiate bi-directional gRPC requests to Datadog. Learn [how to configure the reverse connection][].

Reviewer 2: Need to add the following to the bottom of the page:
```
[1]: /cloudprem/configure/ingress/
[2]: /cloudprem/configure/reverse_connection/
```

> - Let CloudPrem initiates a bi-directional gRPC requests: this is the reverse connection feature.  Learn how to configure the reverse connection [here]()

Reviewer 3: links not filled

> - Let CloudPrem initiates a bi-directional gRPC requests: this is the reverse connection feature.  Learn how to configure the reverse connection [here]()

Reviewer: ```suggestion
- Let CloudPrem initiate a bi-directional gRPC request: This is the reverse connection feature. Learn how to configure the [reverse connection][].
```

> - Let CloudPrem initiates a bi-directional gRPC requests: this is the reverse connection feature.  Learn how to configure the reverse connection [here]()

Reviewer: ```suggestion
- **Reverse connection**: Let CloudPrem initiate bi-directional gRPC requests to Datadog. Learn [how to configure the reverse connection][].
```
